sdk/eventhub/Azure.Messaging.EventHubs/design/proposal-event-hubs-first-preview.md

```diff
@@ -0,0 +1,564 @@
+# .NET Event Hubs Client: Track Two Proposal (First Preview)
+
+## Summary

```

Comment:
No content review needed;  this is an existing design proposal being preserved here by request of the architecture board.

Reply:
:see_no_evil: